src/main/java/com/google/javascript/gents/ModuleConversionPass.java

```diff
@@ -280,13 +288,28 @@ public boolean shouldTraverse(NodeTraversal t, Node n, Node parent) {
    *
    * <pre>
    *   import localName from "goog:old.namespace.syntax";
-   *   import {A as localName} from "./valueExports";
-   * import * as localName from "./objectExports";
-   * import "./sideEffectsOnly"
+   *   import {A as localName, B} from "./valueExports";
+   *   import * as localName from "./objectExports";
+   *   import "./sideEffectsOnly"
    * </pre>
    */
   void convertRequireToImportStatements(
-      Node n, String fullLocalName, String requiredNamespace, boolean isDestructuringImports) {
+      Node n,
+      List<String> fullLocalNames,
+      String requiredNamespace,
+      boolean isDestructuringImports) {
+    // The rest of the functions assume that fullLocalNames contains one and only one element if
+    // this is a destructuring import.
```

Comment:
do you mean 'if this is not a destructuring import'?

If so, that is ok because it is invalid ES6 to have `import * as foo, * as bar` or to have anything but `*` on the LHS.

Reply:
Otherwise, also ok, but file a bug because it is possible to have 'import {foo as bar, buz as buzz} from ' ...

Reply:
Yeah that's what I meant (I missed "not").